mautic/mautic
# Add support for {today} token

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 
| New feature/enhancement? (use the a.x branch)      | 🚀 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              |  🟢 
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/343
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | 



#### Description:



#### Steps to test this PR:

This PR added `{today}` token support for emails and pages. 


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create email  and landing page and test few variations from documentation https://github.com/mautic/mautic-documentation/pull/343
3. See If everything work as expected
